### PR TITLE
Remove all mentions of EmbedTest262ReportResultsTable

### LINKED ISSUE #297: Remove all mentions of EmbedTest262ReportResultsTable
See https://github.com/mdn/content/pull/3375/files

We need to replicate this on the translated content. Examples:

- [x] [ja](https://github.com/mdn/translated-content/blob/a4f49f9a14007bf65efdbe671d6383e8b233ac5b/files/ja/web/javascript/reference/global_objects/bigint/index.html#L286-L290)
- [x] [es](https://github.com/mdn/translated-content/blob/a4f49f9a14007bf65efdbe671d6383e8b233ac5b/files/es/web/javascript/reference/lexical_grammar/index.html#L647-L651)
- [x] [de](https://github.com/mdn/translated-content/blob/a4f49f9a14007bf65efdbe671d6383e8b233ac5b/files/de/web/javascript/reference/global_objects/globalthis/index.html#L86-L90)
- [x] [fr](https://github.com/mdn/translated-content/blob/a4f49f9a14007bf65efdbe671d6383e8b233ac5b/files/fr/web/javascript/reference/operators/nullish_coalescing_operator/index.html#L139-L143)
- [x] [zh-CN](https://github.com/mdn/translated-content/blob/a4f49f9a14007bf65efdbe671d6383e8b233ac5b/files/zh-cn/web/javascript/reference/statements/import.meta/index.html#L93-L97)

### PR DESCRIPTION
**Pull request by peterbe**
Part of #297

I basically searched (and destroyed) on all the locales we don't have an active community.

### CODE REVIEW

**Comment by peterbe** [comment]
@tristantheb Since you're now an expert at killing these (in French), would you mind reviewing this one too?

**Comment by tristantheb** [comment]
I added all the files detected by the search, this time we have them all ^^

**Review by tristantheb — APPROVED** [review]
The Deepl/GG Trad check of the various files seems to indicate that the right pieces have been deleted

File: files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html [hunk 1]
```diff
@@ -229,13 +229,6 @@ <h2 id="Browser_compatibility" name="Browser_compatibility">ブラウザーの
 
 <p>{{Compat("javascript.builtins.Intl.DateTimeFormat.DateTimeFormat")}}</p>
 
-<h3 id="Implementation_Progress" name="Implementation_Progress">実装の進捗状況</h3>
-
-<p>以下の表は、クロスブラウザーの安定性にまだ達していない新機能の日々の実装状況を示しています。このデータは、 JavaScript の標準テストスイートである <a href="https://github.com/tc39/test262">Test262</a> で関連する機能テストをナイトリービルド、または各ブラウザの JavaScript エンジンの最新リリースで実行することで生成されます。</p>
-
-<div>{{EmbedTest262ReportResultsTable("Intl.DateTimeFormat-datetimestyle")}}</div>
-</div>
```
Comment (peterbe) [inline files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html:237]: Here you've deleted 2 `</div>` but only `<div>`.
Is this page working now? Was it always busted perhaps? Can you take a look?

File: files/zh-cn/web/javascript/reference/global_objects/bigint/index.html [hunk 2]
```diff
@@ -253,17 +253,9 @@ <h2 id="标准">标准</h2>
...
 
-<p>{{Compat("javascript.builtins.BigInt")}}</p>
-
-<h3 id="实施进度">实施进度</h3>
-
-<p>下表提供了此功能的每日实现状态，因为此功能尚未达到跨浏览器稳定性。数据是通过在 Test262 中运行相关的特性测试生成的，<a href="https://github.com/tc39/test262">Test262</a> 是 JavaScript 的标准测试套件，在夜间构建，或者是每个浏览器的 JavaScript 引擎的最新版本中运行。</p>
-
-<p>{{EmbedTest262ReportResultsTable("BigInt")}}</p>
-</div>
+<p>{{Compat("javascript.builtins.BigInt")}}</p
```
Comment (peterbe) [inline files/zh-cn/web/javascript/reference/global_objects/bigint/index.html:258]:
```suggestion
<p>{{Compat("javascript.builtins.BigInt")}}</p>
```

File: files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html (hunk 1 above)
Comment (peterbe) [reply files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html:237]: I tested http://localhost:5000/ja/docs/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat and it's looking fine.

**Comment by peterbe** [comment]
I've tested most of the pages now. They render fine. 

I think this is ready to merge.

File: files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html (hunk 1 above)
Comment (tristantheb) [reply files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html:237]: I'm going to check the file for the number of times the <div> are opened anyway to make sure it's not just the automatic closing of the browser that would give a false positive

Comment (tristantheb) [reply files/ja/web/javascript/reference/global_objects/intl/datetimeformat/datetimeformat/index.html:237]: Founded a unused `<div>` on the top of the content, it's ok now ^^

**Comment by tristantheb** [comment]
It's ok for the `</div>` problem 👀

**Comment by peterbe** [comment]
Let's just merge this.

**Pull request merged**